### PR TITLE
replaced sys with subprocess

### PR DESCRIPTION
**Pull request by kpoularakis**
## Description

Please read our [CONTRIBUTING.md](https://github.com/pytorch/serve/blob/master/CONTRIBUTING.md) prior to creating your first pull request.

Please include a summary of the feature or issue being fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

Please describe the Unit or Integration tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
Logs for Test A

- [ ] Test B
Logs for Test B

## Checklist:

- [ ] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Comment by msaroufim** [comment]
Hi @kpoularakis the workflow in git/github is commit based so you can update code to reflect new changes. Having a Pull Request for every version of your code makes it confusing to keep track of history and code review

File: ts_scripts/sanity_utils.py [hunk 1]
```diff
@@ -19,7 +20,7 @@ def run_markdown_link_checker():
     for mdfile in glob.glob("**/*.md", recursive=True):
         cmd = f"markdown-link-check {mdfile} --config link_check_config.json"
         print(f"## In directory: {os.getcwd()} | Executing command: {cmd}")
-        status = os.system(cmd)
+        status = subprocess.run(cmd)
```
Comment (maaquib) [inline ts_scripts/sanity_utils.py:23]: The return types of `subprocess.run` and `os.system` are different. Please change the status code check logic. https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess

**Review by maaquib — CHANGES_REQUESTED** [review]
The return types of `subprocess.run` and `os.system` are different. Please change the status code check logic so that the ci tests would pass. https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess

**Comment by agunapal** [comment]
@kpoularakis Could you please update the PR with why this change is being made. Thanks

**Pull request closed without merging**